content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
 with the purchase order. Clicking on the :guilabel:`Receipt` smart button will show the receipt
 order.
 
-.. image:: receipts_three_steps/three-step-purchase-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-smart-button.png
    :align: center
    :alt: After confirming a purchase order, a Receipt smart button will appear.
 
 Process a receipt
 -----------------
 
-One receipt and two internal transfers (one transfer to quality, and a subsequent transfer to stock)
-will be created once the purchase order is confirmed. To view these transfers, go to
-:menuselection:`Inventory --> Operations --> Transfers`.
+Once the purchase order is confirmed, a receipt (`WH/IN`) operation is generated and ready to
```

Comment:
You could also use an abbreviation tag for "purchase order" if this isn't the first time the term has been mentioned in the doc.
```suggestion
Once the purchase order (PO) is confirmed, a receipt (`WH/IN`) operation is generated and ready to
```

Reply:
Suggestion implemented ✅ - I'll definitely add in a replace abbr tag for all the POs throughout the doc in a full rewrite PR to follow!